templates/repo/issue/labels/label_edit_modal.tmpl

```diff
@@ -1,22 +1,13 @@
-<div class="ui g-modal-confirm delete modal">
-	<div class="header">
-		{{svg "octicon-trash"}}
-		{{ctx.Locale.Tr "repo.issues.label_deletion"}}
-	</div>
-	<div class="content">
-		<p>{{ctx.Locale.Tr "repo.issues.label_deletion_desc"}}</p>
-	</div>
-	{{template "base/modal_actions_confirm" .}}
-</div>
-
-<div class="ui small edit-label modal">
-	<div class="header">
-		{{ctx.Locale.Tr "repo.issues.label_modify"}}
-	</div>
+<div class="ui small modal" id="issue-label-edit-modal"
+		data-repo-link="{{$.Link}}"
```

Comment:
this is not repo link, something like `/user/repo/labels` if you want to chage it

otherwise, i tested and everything is working

Reply:
Good catch, you're right, I should rename it ....